File: Java Fundamentals/Methods/src/FactorialDivision.java
import java.util.Scanner;

public class FactorialDivision {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int firstNum = Integer.parseInt(scan.nextLine());
        int secondNum = Integer.parseInt(scan.nextLine());

        double firstFactorial = getFactorial(firstNum);
        double secondFactorial = getFactorial(secondNum);

        double result = (double) firstFactorial / secondFactorial;

        System.out.printf("%.2f",result);
    }

    public static double getFactorial(double num){
        double factorial = 1;

        for (double i = num; i >= 1; i--) {
            factorial *= i;
        }
        return factorial;
    }
}

File: JS Front-End/Functions and Statements/FactorialDivision.js
function printDivisionOfFactorialsOf(firstNum, secondNum){

    function getFactorial(num){
        if(num <= 1){
            return 1;
        }

        return num * getFactorial(num - 1);
    }

    let result = getFactorial(firstNum) / getFactorial(secondNum);
    console.log(result.toFixed(2));
}

printDivisionOfFactorialsOf(5, 2);
printDivisionOfFactorialsOf(6, 2);